### DISCUSSION TITLE
Light effects drop-down location

### DISCUSSION THREAD

**Discussion by etamtlosz** [home-assistant/frontend#7379] [Dashboard]
The light effects drop-down being on the bottom of the card, there can be an issue seeing/showing the whole effect list. Usually there are only 2 rows visible.
I would like to recommend moving it from the bottom of the card somewhere to the top, for example above the color wheel.
![lovelaceui](https://user-images.githubusercontent.com/7436784/96357070-fdeb9f80-1152-11eb-970f-c7fff73327bf.jpg)

**Comment by spacegaier** [comment]
Will be solved by PR https://github.com/home-assistant/frontend/pull/7956.
